values.yaml

```diff
@@ -547,6 +547,7 @@ redis:
   # ## https://github.com/bitnami/charts/tree/master/bitnami/redis#master-slave-with-sentinel
   sentinel:
     enabled: true
+    staticID: true
```

Comment:
Would be nice to include a small code comment about this new value, perhaps with the original var description and maybe a bit with the context of how it helps in this situation.

Reply:
Hi Does this looks fine now?